shared/style/sk_key_utils.css

```diff
@@ -0,0 +1,379 @@
+.sk-button-container {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background-color: #666;
+  color: #fff;
+  z-index: 99;
+  content: "";
+  clear: both;
+  display: block;
+  text-align: right;
+}
+
+.sk-hide-button {
+  display: none;
+}
+
+.sk-buttons {
+  float: left;
+  padding: 1rem;
+  width: 33.33%;
+  font-size: 1rem;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  -ms-text-overflow: ellipsis;
+  -o-text-overflow: ellipsis;
+  text-overflow: ellipsis;
+}
+
+.sk-button-container .sk-buttons.LSK {
+  text-align: left;
+}
+
+.sk-button-container .sk-buttons.CSK {
+  text-align: center;
+}
+
+.sk-button-container .sk-buttons.CSK:first-child {
+  margin-left: 33.33%;
+}
+
+.sk-button-container .sk-buttons.RSK {
+  float: right;
+  text-align: right;
+}
+
+.sk-cancel-key {
+  color: #fff;
+  font-size: 1rem;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 1rem;
+  background: #666;
+}
+
+form[role="dialog"][data-subtype="menu"] {
+  visibility: hidden;
+}
+
+form[role="dialog"][data-subtype="menu"].visible {
+  visibility: visible;
+}
+
+/* Action Menu 
+ * Copied from shared/style/action_menu.css 
+ * Some cleanup needs to be done
+*/
```

Comment:
This would all probably be better in a separate CSS file, I think.  Ideally just document that this module requires shared/style/action_menu.css.  Or customize it and create a simplified context_menu.css file that has only what we need here.